corehq/apps/reports/views.py

```diff
@@ -1065,6 +1067,11 @@ def _product_name(product_id):
         repeat_records = get_repeat_records_by_payload_id(self.domain, self.case_id)
 
         can_edit_data = self.request.couch_user.can_edit_data
+        show_properties_edit = (
+            can_edit_data
+            and has_privilege(self.request, privileges.DATA_CLEANUP)
+            and not IS_ICDS_ENVIRONMENT
```

Comment:
we are blocking the views by usign domain specific restriction through `check_data_interfaces_blocked_for_domain` but hidding links for all the environment, we should keep it the same at both times, so this i believe should use `is_icds_cas_project` instead.